tests/test_source.py

```diff
@@ -0,0 +1,101 @@
+from sys import version_info
+from typing import Type
+
+if version_info.major == 3 and version_info.minor < 8:
+    from mock import AsyncMock, call, patch
+else:
+    from unittest.mock import call, patch, AsyncMock
+
+import pytest
+
+from metricq.source import Source
+from metricq.source_metric import SourceMetric
+
+pytestmark = pytest.mark.asyncio
+
+
+class _TestSource(Source):
+    async def task(self):
+        assert False, "This should not be run"
+
+    rpc: AsyncMock
+
+
+@pytest.fixture
+def source():
+
```

Comment:
```suggestion
```

Reply:
B-b-b-but... `black` tells me that the empty line is correct :(